packages/test/test-end-to-end-tests/src/test/sharedInterval.spec.ts

```diff
@@ -367,7 +367,8 @@ describeCompat("SharedInterval", "NoCompat", (getTestObjectProvider, apis) => {
 			assertIntervals([{ start: 0, end: 1 }]);
 		});
 
-		it("repeated replacement", async () => {
+		// 277870, 278093, 281244
```

Comment:
This test seems to be very simple and only has 1 await, so might be a good candidate for investigating if something might be hanging or taking too long during `await provider.ensureSynchronized()`.